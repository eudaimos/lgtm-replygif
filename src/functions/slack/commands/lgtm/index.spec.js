import { lgtm } from '.';

describe('[Function: lgtm]', () => {
  it('should return the expected response', async () => {
    expect(await lgtm()).toMatchSnapshot();
  });
});
