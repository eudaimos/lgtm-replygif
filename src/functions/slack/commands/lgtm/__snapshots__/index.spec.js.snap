// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Function: lgtm] should return the expected response 1`] = `
Object {
  "body": "{\\"message\\":\\"Go Serverless v1.0! Your function executed successfully!\\",\\"version\\":\\"v12.21.0\\"}",
  "statusCode": 200,
}
`;
